clients/privacy-center/pages/index.tsx

```diff
@@ -187,6 +188,13 @@ const Home: NextPage = () => {
               {paragraph}
             </Text>
           ))}
+          {config.privacy_policy_visible &&
+          config.privacy_policy_url &&
+          config.privacy_policy_url_text ? (
+            <Link href={config.privacy_policy_url} isExternal>
+              {config.privacy_policy_url_text}
+            </Link>
+          ) : null}
```

Comment:
Why also include `privacy_policy_visible`, shouldn't we just best infer that from the other configurable params?
```
config.privacy_policy_url && config.privacy_policy_url_text ? ( ... )
```

Reply:
Additionally, some design feedback: this "Privacy Policy" link just feels... orphaned and out of place. Can we make it more intuitive? Maybe it feels better if we move this link below the action cards?

![image](https://github.com/ethyca/fides/assets/1834295/f3c0b453-4efe-43f4-bab1-35df4aebc7ec)


Let's get some design feedback and ship it!

Reply:
Pushed an update to this:
<img width="1088" alt="image" src="https://github.com/ethyca/fides/assets/1834295/50ecea9e-d030-4059-8223-6cae07093c13">

I think this is solid now so I'd say ship it

Reply:
Thanks for cleaning up that initial ask! This is much sharper ⭐

Reply:
Resolved in https://github.com/ethyca/fides/pull/4090/commits/82a2f7e90d57342210fde580320ab9f8acd8b07d